src/domain/errorPage/ErrorPage.module.scss

```diff
@@ -0,0 +1,33 @@
+@import 'variables';
+
+.container {
+  --header-fontsize: var(--fontsize-h-2);
+}
+
+.container {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: var(--spacing-4-xl);
+
+  .iconWrapper {
+    width: 3rem;
+  }
+
+  h1 {
+    font-size: var(--header-fontsize);
+    margin: var(--spacing-l) 0;
+  }
+
+  .description {
+    max-width: 550px;
```

Comment:
Does this break layout on mobile devices? I'm not sure

Reply:
I don't thinks so, it is only max-width. But the container needs some left and right padding for the mobile width